GitUpKit/Views/Base.lproj/GIMapViewController.xib

```diff
@@ -111,6 +111,12 @@ DQ
                         <action selector="editSelectedCommitMessage:" target="-2" id="PrK-v7-e6I"/>
                     </connections>
                 </menuItem>
+                <menuItem title="Copy Message…" alternate="YES" keyEquivalent="e" id="LnA-LI-dEb">
```

Comment:
`...` is reserved for actions where there will be something the user has to follow up on. There is no followup dialog here so the text should simply be `Copy Message`.